crypto/bech32/bech32_test.go

```diff
@@ -0,0 +1,37 @@
+package bech32
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+)
+
+func TestBench32EncodeDecode(t *testing.T) {
```

Comment:
We could use a few more (table) tests here...

At least grab all `Decode` tests from here... https://github.com/sipa/bech32/blob/master/ref/go/src/bech32/bech32_test.go to feel certain it works well.

Reply:
I have added table tests https://github.com/iov-one/weave/pull/415/commits/b4fd343702ee9ed2b4918374384ed0c14c32b6e6

I would prefer not to copy code from that bech32 implementation as it is using weird types. I think the author does not know the difference between `int` and `byte` in Go :see_no_evil: